backend/operation.mli

```diff
@@ -167,6 +167,15 @@ type t =
         dbginfo : Cmm.alloc_dbginfo;
         mode : Cmm.Alloc_mode.t
       }
+  | External of
```

Comment:
As discussed off-line, we may want to
use a different name to hint at the
no `caml_c_call` / no divergence
property. (Could of course happen in
a follow-up pull request.)